src/components/TipInput.vue

```diff
@@ -130,25 +130,33 @@ export default {
       return (this.value > 0 && (this.isRetip || this.isMessageValid)) || this.isMessageValid;
     },
     isTipped() {
-      return !this.loading
-        || (this.tip.sender === this.account)
-        || this.tip.retips.filter((retip) => retip.sender === this.account).length > 0;
+      if (!this.stats || !this.derivedTipStats) {
+        return false;
+      }
+      return this.derivedTipStats.senders.find((sender) => sender === this.account);
     },
     iconTip() {
       return this.isTipped ? iconTipped : iconTip;
     },
+    derivedTipStats() {
+      if (!this.stats || !this.stats.by_url) {
+        return 0;
```

Comment:
this should probably return `null` as it is compatible with the other return results (which return object or null)